protobuf/src/stream.rs

```diff
@@ -790,14 +790,14 @@ impl<'a> CodedOutputStream<'a> {
                 self.position = 0;
             },
             OutputTarget::Bytes => {
-                panic!("refresh_buffer must not be called on CodedOutputStream create from slice");
```

Comment:
Can you please split the patch into changes of logic and typo fixes? I appreciate doc fixes, and if you can't do it, I'll split the patch myself.

Reply:
changes moved here: https://github.com/stepancheg/rust-protobuf/pull/375